test/tests/mesh/node_list_from_side_list/displace_node_list.i

```diff
@@ -0,0 +1,7 @@
+[Mesh]
+  [gen]
+    type = FileMeshGenerator
+    file = bcid_conflict.xda
```

Comment:
looks like you forgot to check in this file

use -f maybe

Reply:
Of course, there's a .gitignore line ignoring .xda.  Why would we ever want to use a file format that supports more features in 30% (or 10% if you gz it) of the size?

(I lash out because I'm embarrassed; thanks for catching this)